File: jdroid-android-firebase-admob/src/main/java/com/jdroid/android/firebase/admob/helpers/AdHelper.java
package com.jdroid.android.firebase.admob.helpers;

import android.app.Activity;
import android.view.ViewGroup;

import com.google.android.gms.ads.AdListener;

import java.util.List;

public interface AdHelper {
	
	AdHelper setAdUnitId(String adUnitId);
	
	AdHelper setAdListeners(List<AdListener> adListeners);

	void loadAd(Activity activity, ViewGroup adViewContainer);
}

File: jdroid-android-firebase-admob/src/main/java/com/jdroid/android/firebase/admob/helpers/AdHelper.kt
package com.jdroid.android.firebase.admob.helpers

import android.app.Activity
import android.view.ViewGroup

import com.google.android.gms.ads.AdListener

interface AdHelper {

    fun setAdUnitId(adUnitId: String): AdHelper

    fun setAdListeners(adListeners: List<AdListener>): AdHelper

    fun loadAd(activity: Activity, adViewContainer: ViewGroup?)
}
